bundles/org.eclipse.swt/Eclipse SWT PI/common/org/eclipse/swt/internal/Library.java

```diff
@@ -35,12 +35,12 @@ public class Library {
 	/**
 	 * SWT Minor version number (must be in the range 0..999)
 	 */
-	static int MINOR_VERSION = 964;
+	static int MINOR_VERSION = 965;
 
 	/**
 	 * SWT revision number (must be >= 0)
 	 */
-	static int REVISION = 8;
+	static int REVISION = 1;
```

Comment:
Because the subsequent build will increment the `REVISION` and rebuild the natives I think it would make sense to start with zero in this scenario.
When using one, the the first revision eventually deployed by I-builds is two.